applications/soupbintcp-gateway/src/main/java/com/paritytrading/nassau/soupbintcp/gateway/Session.java

```diff
@@ -41,11 +38,7 @@ public void heartbeatTimeout(SoupBinTCPServer session) throws IOException {
 
     @Override
     public void loginRequest(SoupBinTCPServer session, SoupBinTCP.LoginRequest payload) throws IOException {
-        System.arraycopy(payload.requestedSession, 0, loginAccepted.session, 0,
-                payload.requestedSession.length);
-        ASCII.putLongRight(loginAccepted.sequenceNumber, 1);
-
-        downstream.accept(loginAccepted);
+        downstream.accept(new SoupBinTCP.LoginAccepted(payload.getSession(), payload.getSequenceNumber()));
```

Comment:
This is no longer re-using the static `LoginAccepted` buffers at the cost of a new allocation on each `LoginRequest` -- it seems like this the right thing to do, though...?